ui/css/utilities/fonts.scss

```diff
@@ -80,3 +80,10 @@ $font-path: './fonts';
   font-weight: 400;
   font-style: normal;
 }
+
+@font-face {
+  font-family: 'MMPoly';
+  src: url('#{$font-path}/MMPoly/MM_Poly-Variable.woff2') format('woff2');
+  font-weight: 400;
+  font-style: normal;
+}
```

Comment:
Defiining the @font-face which matches the latest [design tokens release](https://github.com/MetaMask/metamask-design-system/blob/main/packages/design-tokens/MIGRATION.md#from-version-600-to-700)